src/modules/ROOT/pages/changelog/202309/05.json

```diff
@@ -6,7 +6,7 @@
   "added": [
     {
       "what": [
-        "the option to access user information like username, handle, email, etc. in pages, charts, and record fields."
+        "the option to access user information from link:https://docs.cortezaproject.org/corteza-docs/2023.9/integrator-guide/expr/type-reference.html#objref-user[User values] in pages, charts, and record fields."
```

Comment:
nono, use an `xref` (global search project to get an example); don't link to a static page as the link could break